poseidon/export_test_vectors/README.md

```diff
@@ -20,17 +20,29 @@ cargo run --bin export_test_vectors --all-features -- <MODE> <PARAM_TYPE> <OUTPU
 
 - `<OUTPUT_FILE>`: Output file path, use `-` for stdout
 
+### Options
+
+- `--format <FORMAT>`: Output file format (default: `json`)
+  - `es5`: ES5 JavaScript format
+  - `json`: JSON format
+
 ### Examples
 
 ```bash
-# Export b10 legacy vectors to a file
+# Export b10 legacy vectors to a JSON file
 cargo run --bin export_test_vectors --all-features -- b10 legacy vectors.json
 
-# Export hex kimchi vectors to stdout
+# Export hex kimchi vectors to stdout in JSON format
 cargo run --bin export_test_vectors --all-features -- hex kimchi -
 
-# Export hex legacy vectors to a file
+# Export hex legacy vectors to a JSON file
 cargo run --bin export_test_vectors --all-features -- hex legacy test_vectors.json
+
+# Export hex kimchi vectors to an ES5 JavaScript file
+cargo run --bin export_test_vectors --all-features -- hex kimchi poseidon-kimchi.js --format es5
```

Comment:
Is this expected that when `kimchi` is chosen you have to pass `poseidon-kimchi.js` as well?

Reply:
No :thinking: